src/main/java/csv/processing/CsvProcessorWrite.java

```diff
@@ -0,0 +1,51 @@
+package csv.processing;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvProcessorWrite {
```

Comment:
```suggestion
public class CsvWriteProcessor {
```

Давай создадим в этом классе только 1 метод, который будет принимать на вход название файла, в который нужно записать информацию и String с самой информацией, которую нужно записать. И лучше создать какой-то общий интерфейс (какой-нибудь DataWriteProcessor) и унаследовать CsvWriteProcessor от него (если в будущем нам понадобиться писать информацию не в файл а куда-то еще, мы сможем просто написать еще одну реализацию DataWriteProcessor для этого)